frontend/packages/console-app/src/components/quick-starts/utils/quick-start-context.ts

```diff
@@ -187,6 +188,23 @@ export const useValuesForQuickStartContext = (): QuickStartContextValues => {
     [setAllQuickStartStates],
   );
 
+  const getQuickStartActiveTask = useCallback(
```

Comment:
I don't think you need this method at all.

you can just use the activeQuickStartState in the component that it is being used to get the activeTaskNumber for the activeQuickStart

Reply:
moved it to QuickStartController and computing activeTask inside controller